README.md

```diff
@@ -1 +1,60 @@
-# ul-image-api
\ No newline at end of file
+# UL Image API
+
+The [Unified Listing](http://ul.gpii.net) is a resource for people who want to find assistive technology solutions and
+mainstream products with assistive features.  This package is used within the Unified Listing to allow:
+
+1. Contributing images and image metadata associated with [Unified Listing](http://ul.gpii.net/) entries.
+2. Reviewing and approving or rejecting contributions.
+3. Allowing users to see the status of their contributions.
+4. Viewing approved images as part of a product listing.
+
+Anyone (even users who are not logged in) can view the images and galleries associated with the "unified" source.  This
+is a curated data source created by reviewers based on imported data and user contributions.
+
+By [signing up for a free account](/api/user/signup) and [logging in](/api/user/login), users gain the ability to
+contribute updates to the data served by this API.
+
+Users can contribute new images, which are only visible to the submitter and reviewers.  An image is approved or
+rejected by a reviewer.
+
+Users can submit metadata regarding an image (for example, to suggest a better description for a given image).
+These suggestions are only visible to the submitter and reviewers.  Reviewers approve and reject contributed metadata.
+
+To contribute image data to the live Unified Listing, please visit [http://ul.gpii.net].  To try out the software
+locally, please read the "Developer Documentation" below.
+
+# Developer Documentation
+
+## The API
+
+The API provides a series of REST inputs.  For details regarding the endpoints, their accepted input, and their return
+values, see [docs/api-rest-docs.md](the API documentation).
+
+The API is built as a series of [Fluid components](http://fluidproject.org/). For more information, see
+[docs/api-component-docs.md](the API component documentation).  In addition, the API is built using the following key
+libraries:
+
+1. [Sharp](http://sharp.dimens.io/), for image resizing.
```

Comment:
Somewhere (perhaps not here) there should be materials for a technology evaluation exercise - what requirements did you have, how did you evaluate them, and what alternatives did you consider for these libraries.